modules/ingestor/src/endpoints/advisory.rs

```diff
@@ -14,11 +24,12 @@ use tokio_util::io::ReaderStream;
 /// Upload a new advisory
 pub async fn upload_advisory(
     service: web::Data<IngestorService>,
+    web::Query(UploadParams { issuer }): web::Query<UploadParams>,
     bytes: web::Bytes,
 ) -> Result<impl Responder, Error> {
     let fmt = Format::from_bytes(&bytes)?;
     let payload = ReaderStream::new(&*bytes);
-    let advisory_id = service.ingest("rest-api", fmt, payload).await?;
+    let advisory_id = service.ingest("rest-api", issuer, fmt, payload).await?;
```

Comment:
So, I was uncomfortable with this first param, "rest-api", which used to be the `location` until we decided that wasn't really useful in #233. So it seems like maybe you're bringing the location concept back in the form of an issuer? If that's the case, let's get rid of the hard-coded "rest-api" arg. Unless you think it's useful. I don't.

Reply:
"remove location" I think should be a separate PR. Here, the issuer is *optional* and only used for OSV where that information is not necessarily derivable from the contents.